apps/daimo-mobile/src/view/screen/send/SendTransferButton.tsx

```diff
@@ -79,14 +80,22 @@ export function SendTransferButton({
   const fullMemo = getFullMemo({ memo, money });
 
   // Pending swap, appears immediately > replaced by onchain data
-  const pendingOpBase = {
+  const pendingOp: TransferSwapClog = {
+    type: "transfer",
     from: account.address,
     to: recipient.addr,
     amount: Number(dollarsToAmount(dollarsStr)),
     memo: fullMemo,
     status: OpStatus.pending,
     timestamp: 0,
   };
+  if (isSwap || isBridge) {
+    pendingOp.postSwapTransfer = {
+      amount: `${BigInt(route?.toAmount || pendingOp.amount)}`,
```

Comment:
* send to ETH, etc: show destination token amount
* (no swap route): show USDC send amount = same as fastCCTP received amount